src/core/src/UsdUtil.cpp

```diff
@@ -370,8 +370,10 @@ pxr::CesiumSession getOrCreateCesiumSession() {
 }
 
 pxr::CesiumGeoreference getOrCreateCesiumGeoreference() {
-    static const auto path = pxr::SdfPath("/CesiumGeoreference");
+    return getCesiumGeoreference(pxr::SdfPath("/CesiumGeoreference"));
+}
 
+pxr::CesiumGeoreference getCesiumGeoreference(const pxr::SdfPath& path) {
```

Comment:
The other `getCesiumXXXX` functions assert if the prim doesn't exist or is invalid. Should this behave the same way? The creation code would move into `getOrCreateCesiumGeoreference`.